### PR TITLE
fix simulation output

### PR DESCRIPTION
**Pull request by bevns**
Looks like the CLI tool was updated and the output is now a little different. The `done` was removed in https://github.com/BattlesnakeOfficial/rules/pull/109/commits/741b6ca1191f00330ced4653db00247bce58d644#diff-07f7fb51e394a858e20eeefeec54f5356d89c9d5697b6559424b274a589a205fL275-R286

This fixes that so we can log the simulation again.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ FROM ruby:3.1.2
 EXPOSE 4567
 CMD ["/usr/bin/supervisord"]
 
-RUN apt-get update && apt-get install --no-install-suggests -y supervisor && apt-get install pip -y
+RUN apt-get update && apt-get install --no-install-suggests -y supervisor && apt-get install pip -y && apt-get install jq -y
```
Comment (djdefi) [inline Dockerfile:6]: Can just add jq after pip and avoid another &&

Comment (bevns) [inline Dockerfile:6]:
```suggestion
RUN apt-get update && apt-get install --no-install-suggests -y supervisor && apt-get install pip jq -y
```

Comment (bevns) [reply Dockerfile:6]: ahh ya true!
```suggestion
RUN apt-get update && apt-get install --no-install-suggests -y supervisor && apt-get install pip jq -y
```

**Comment by bevns** [comment]
I think I prefer seeing the older output where it states the number of turns etc after each run but this is working and could be merged like this.

**Comment by bevns** [comment]
@djdefi how do you like this output? We could easily manipulate each json string instead into something we like better. ¯\_(ツ)_/¯

**Comment by djdefi** [comment]
Fine for now :shipit:

**Review by djdefi — APPROVED** [review]

**Pull request merged**